client/components/App/index.jsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { renderRoutes } from 'react-router-config';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import queryString from 'query-string';
+import { handleLogin } from '../../actions/login';
 
-const App = ({ route }) => {
-    return (
-        <div>
-            <h1>ARIA-AT REPORT</h1>
-            {renderRoutes(route.routes)}
-        </div>
-    );
-};
+class App extends Component {
+    componentDidMount() {
+        const { location, dispatch } = this.props;
+        const { login } = queryString.parse(location.search);
+        if (login) {
+            dispatch(handleLogin());
+        }
+    }
+    render() {
+        const { route } = this.props;
+        return (
+            <div>
+                <h1>ARIA-AT REPORT</h1>
+                <Link to="/login">Login</Link>
+                <br></br>
+                <Link to="/account/settings">Settings</Link>
+                {renderRoutes(route.routes)}
```

Comment:
So when a route is update, this is replace with what is specified in `client/routes/index.js`?